lib/gui/app.js

```diff
@@ -176,6 +176,11 @@ app.run(() => {
     ].join(' '));
 
     windowProgress.set(currentFlashState.percentage);
+
+    const DASH_UNICODE_CHAR = '\u2013';
```

Comment:
Should this constant be moved to a 'higher level', or does it make sense to keep it so localised?